src/recording/recordingmanager.cpp

```diff
@@ -41,6 +41,29 @@ RecordingManager::RecordingManager(UserSettingsPointer pConfig, EngineMixer* pEn
     m_split_size = getFileSplitSize();
     m_split_time = getFileSplitSeconds();
 
+    // reads mixxx.cfg, else return default value = 44100
+    auto recSampleRateInitVal = pConfig->getValue(
+            ConfigKey(RECORDING_PREF_KEY, "rec_samplerate"),
+            44100.0);
+    qDebug() << "rec sr init val: " << recSampleRateInitVal;
+
+    m_pRecSampleRate = std::make_unique<ControlObject>(
+            ConfigKey(RECORDING_PREF_KEY, QStringLiteral("rec_samplerate")),
+            false,
+            true,
+            static_cast<double>(recSampleRateInitVal));
+    m_pRecSampleRate->set(recSampleRateInitVal);
+
+    auto useEngineSampleRateInitVal = pConfig->getValue(
+            ConfigKey(RECORDING_PREF_KEY, "use_engine_samplerate"),
+            0);
+
+    m_pUseEngineSampleRate = std::make_unique<ControlObject>(
+            ConfigKey(RECORDING_PREF_KEY, QStringLiteral("use_engine_samplerate")),
+            false,
+            true,
+            static_cast<bool>(useEngineSampleRateInitVal));
```

Comment:
I think we should default to the engine samplerate, because this is the default in the cersiont version. 

```suggestion
    bool useEngineSampleRateInitVal = pConfig->getValue(
            ConfigKey(RECORDING_PREF_KEY, "use_engine_samplerate"),
            true);

    m_pUseEngineSampleRate = std::make_unique<ControlObject>(
            ConfigKey(RECORDING_PREF_KEY, QStringLiteral("use_engine_samplerate")),
            false,
            true,
            useEngineSampleRateInitVal);
```